assets/js/theme/common/carousel/utils/tooltipSetup.js

```diff
@@ -0,0 +1,30 @@
+const carouselTooltipClass = 'carousel-tooltip';
+const carouselTooltip = `<span class="${carouselTooltipClass}"></span>`;
+
+const setupTooltipAriaLabel = ($node) => {
+    const $existedTooltip = $node.find(`.${carouselTooltipClass}`);
+
+    if ($existedTooltip.length) {
+        $existedTooltip.attr('aria-label', $node.attr('aria-label'));
+    } else {
+        const $tooltip = $(carouselTooltip).attr('aria-label', $node.attr('aria-label'));
+        $node.append($tooltip);
+    }
+};
+
+const arrows = (...arrowNodes) => {
+    arrowNodes.forEach($arrow => setupTooltipAriaLabel($arrow));
+};
+
+const dots = ($dots) => {
+    $dots.children().each((idx, dot) => setupTooltipAriaLabel($(dot).find('button')));
+};
+
+export default ($prevArrow, $nextArrow, $dots) => {
```

Comment:
Could we move this file to theme/common/aria folder?

Reply:
We have all carousel utils in this folder. Any reasons we should replace this particular util?